pages/common/cargo-yank.md

```diff
@@ -0,0 +1,17 @@
+# cargo yank
+
+> Remove a pushed crate from the index. This should only be used when you accidentally release a significantly broken crate.
+> Note: this does not remove any data. The crate is still present after a yank - this just prevents new users from downloading it.
```

Comment:
```suggestion
> Note: this does not remove any data. The crate is still present after a yank - this just prevents new projects from using it.
```

Previously it sounded like only newly registered users will be unable to download the crate.